res/css/_common.scss

```diff
@@ -318,6 +318,8 @@ input[type=text]:focus, input[type=password]:focus, textarea:focus {
 .mx_Dialog_lightbox .mx_Dialog_background {
     opacity: $lightbox-background-bg-opacity;
     background-color: $lightbox-background-bg-color;
+    animation-name: mx_Dialog_lightbox_background_keyframes;
```

Comment:
Please handle `@media (prefers-reduced-motion)`, so that we don't animate things when user doesn't want reduced motion.